src/pages/QuizApp/Quiz.js

```diff
@@ -0,0 +1,26 @@
+import PageHeader from "../../Styled.jsx";
+import { QuizContainer} from '../../Styled';
+import QuizCardComp from './QuizCardComp.jsx';
+
+const cardWidth = '500px';
+const cardMinHeight = '300px'
+const buttonBackgroundColour = 'white';
+const answerButtonWidth = '210px';
+const endButtonWidth = '245px';
+
+export default function Quiz() {
+    return (
+	<>
+		<PageHeader>Quiz App</PageHeader>
+    		<QuizContainer>
+				<QuizCardComp
```

Comment:
If I want to give a different set of questions to this QuizCardComp, how would I do this?